docs/rules/spaced-comment.md

```diff
@@ -41,6 +41,12 @@ The `"markers"` array will apply regardless of the value of the first argument,
 The difference between a marker and an exception is that a marker only appears at the beginning of the comment whereas
 exceptions can occur anywhere in the comment string.
 
+You can also define separate exceptions and markers for block and line comments:
+
+```js
+"spaced-comment": [2, "always", { "line": { "markers": ["/"] }, "block": { "markers": ["*"] } }]
```

Comment:
You can also do `exceptions` for each, correct? We should list out all possible options in this example (and maybe use some formatting to spread out across a few lines?).